datasets/kor_nlu/kor_nlu.py

```diff
@@ -0,0 +1,153 @@
+"""Korean Dataset for NLI and STS"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+
+import pandas as pd
+
+import datasets
+
+
+_CITATAION = """\
+    @article{ham2020kornli,
+  title={KorNLI and KorSTS: New Benchmark Datasets for Korean Natural Language Understanding},
+  author={Ham, Jiyeon and Choe, Yo Joong and Park, Kyubyong and Choi, Ilji and Soh, Hyungjoon},
+  journal={arXiv preprint arXiv:2004.03289},
+  year={2020}
+}
+"""
+
+_DESCRIPTION = """\
+    The dataset contains data for bechmarking korean models on NLI and STS
+"""
+
+_URL = "https://github.com/kakaobrain/KorNLUDatasets"
+
+_DATA_URLS = {
+    "nli": {
+        # 'mnli-train': 'https://raw.githubusercontent.com/kakaobrain/KorNLUDatasets/master/KorNLI/multinli.train.ko.tsv',
+        "snli-train": "https://raw.githubusercontent.com/kakaobrain/KorNLUDatasets/master/KorNLI/snli_1.0_train.ko.tsv",
+        "xnli-dev": "https://raw.githubusercontent.com/kakaobrain/KorNLUDatasets/master/KorNLI/xnli.dev.ko.tsv",
+        "xnli-test": "https://raw.githubusercontent.com/kakaobrain/KorNLUDatasets/master/KorNLI/xnli.test.ko.tsv",
+    },
+    "sts": {
+        "train": "https://raw.githubusercontent.com/kakaobrain/KorNLUDatasets/master/KorSTS/sts-train.tsv",
+        "dev": "https://raw.githubusercontent.com/kakaobrain/KorNLUDatasets/master/KorSTS/sts-dev.tsv",
+        "test": "https://raw.githubusercontent.com/kakaobrain/KorNLUDatasets/master/KorSTS/sts-test.tsv",
+    },
+}
+
+
+class KorNluConfig(datasets.BuilderConfig):
+    """BuilderConfig for korNLU"""
+
+    def __init__(self, description, data_url, citation, url, **kwargs):
+        """
+        Args:
+            description: `string`, brief description of the dataset
+            data_url: `dictionary`, dict with url for each split of data.
+            citation: `string`, citation for the dataset.
+            url: `string`, url for information about the dataset.
+            **kwrags: keyword arguments frowarded to super
+        """
+        super(KorNluConfig, self).__init__(version=datasets.Version("1.0.0", ""), **kwargs)
+        self.description = description
+        self.data_url = data_url
+        self.citation = citation
+        self.url = url
+
+
+class korNlu(datasets.GeneratorBasedBuilder):
+    BUILDER_CONFIGS = [
+        KorNluConfig(name=name, description=_DESCRIPTION, data_url=_DATA_URLS[name], citation=_CITATAION, url=_URL)
+        for name in ["nli", "sts"]
+    ]
+
+    def _info(self):
+        features = {}
+        if self.config.name == "nli":
+            labels = ["entailment", "neutral", "contradiction"]
+            features["sentence1"] = datasets.Value("string")
+            features["sentence2"] = datasets.Value("string")
```

Comment:
could you rename them `premise` and `hypothesis` for consistency with other NLI datasets ?

Reply:
made this change